static/templates/settings/organization_settings_admin.hbs

```diff
@@ -142,17 +142,12 @@
                   is_checked=realm_mandatory_topics
                   label=admin_settings_label.realm_mandatory_topics}}
 
-                <div class="input-group">
-                    <label class="checkbox">
-                        <input type="checkbox" class="realm_enable_read_receipts inline-block setting-widget prop-element" name="realm_enable_read_receipts" data-setting-widget-type="boolean" id="id_realm_enable_read_receipts" {{#if realm_enable_read_receipts}}checked="checked"{{/if}}/>
-                        <span></span>
-                    </label>
-                    <label for="id_realm_enable_read_receipts" class="inline realm_enable_read_receipts_label" id="id_realm_enable_read_receipts_label">
-                        {{#tr}}
-                            Enable read receipts (<i>Users can always disable their personal read receipts.</i>)
-                        {{/tr}}
-                    </label>
-                </div>
+                {{> settings_checkbox
+                  setting_name="realm_enable_read_receipts"
+                  prefix="id_"
+                  is_checked=realm_enable_read_receipts
+                  label=admin_settings_label.realm_enable_read_receipts
+                  label_parens_text="Users can always disable their personal read receipts."}}
```

Comment:
Translation tags always need to appear next to the string itself, not run on a variable; the problem is that the process for discovering what translateable strings exist to send to translators does not actually run the code, it just parses the syntax, so it won't know what values are passed.  

Probably the simplest fix is to do `label_parens_text=admin_settings_label.realm_enable_read_receipts_parens_text`, and define that in the JS file; it'll be nice anyway to have the two label parts next to each other in the source.

https://zulip.readthedocs.io/en/latest/translating/internationalization.html should hopefully discuss this detail; if not we should add it.